boards/arm/stm32h747i_disco/stm32h747i_disco_m7.dts

```diff
@@ -54,3 +54,23 @@
 	current-speed = <115200>;
 	status = "okay";
 };
+
+/* From UM2411 Rev 4:
+   With the default setting, the Ethernet feature is not working because of
+   a conflict between ETH_MDC and SAI4_D1 of the MEMs digital microphone */
+   &mac {
+	status = "okay";
+	pinctrl-0 = <&eth_ref_clk_pa1
+		     &eth_mdio_pa2
+		     &eth_crs_dv_pa7
+		     &eth_mdc_pc1
+		     &eth_rxd0_pc4
+		     &eth_rxd1_pc5
+		     &eth_tx_en_pg11
+		     &eth_txd0_pg13
+		     &eth_txd1_pg12>;
```

Comment:
I am not ethernet expert, but don't we need PE2 ETH_nINT signal ?

Reply:
We actually don't need it, at least for other H7 board with ethernet that's totally fine.

Reply:
I can confirm @lochej words: other boards also with RMII working fine without `PE2 ETH_nINT` signal.